packages/@react-aria/landmark/docs/useLandmark.mdx

```diff
@@ -0,0 +1,110 @@
+{/* Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-aria/landmark';
+import {HeaderInfo, FunctionAPI, PageDescription} from '@react-spectrum/docs';
+import {Keyboard} from '@react-spectrum/text';
+import packageData from '@react-aria/landmark/package.json';
+
+---
+category: Navigation
+keywords: [landmarks, aria]
+---
+
+# useLandmark
+
+<PageDescription>{docs.exports.useLandmark.description}</PageDescription>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['useLandmark']}
+  sourceData={[
+    {type: 'W3C', url: 'https://www.w3.org/WAI/ARIA/apg/practices/landmark-regions/'}
+  ]} />
+
+## API
+
+<FunctionAPI function={docs.exports.useLandmark} links={docs.links} />
+
+## Features
+
+Landmarks provide a way to designate important subsections of a page. They allow screen reader users to get an overview of the various sections of the page, and jump to a specific section.
+By default, browsers do not provide a consistent way to navigate between landmarks using the keyboard.
+The `useLandmark` hook enables keyboard navigation between landmarks, and provides a consistent experience across browsers.
+
+* <Keyboard>F6</Keyboard> and <Keyboard>Shift+F6</Keyboard> key navigation between landmarks
+* <Keyboard>Alt+F6</Keyboard> key navigation to the main landmark
+* Support for navigating nested landmarks
+* Support for navigating across iframes
```

Comment:
doesn't this require extra work? what is that extra work?

Reply:
A real iframe example may be difficult in our docs, so I could add a dummy example that doesn't render anything.

Reply:
I think if we know what work is needed for support within iframes, we can just write about it, no need to create a full blown example. I just thought @devongovett had to do some extra work here when he implemented it in another project.

Reply:
Ended up removing it since it was a bit complex.